src/tmpfiles/tmpfiles.c

```diff
@@ -198,6 +198,7 @@ static bool arg_user = false;
 static OperationMask arg_operation = 0;
 static bool arg_boot = false;
 static PagerFlags arg_pager_flags = 0;
+static int arg_factory_reset = -1;
```

Comment:
Why this is tristate, instead of boolean?? Only for logging??

Reply:
See top of this comment: https://github.com/systemd/systemd/pull/24983#discussion_r994341848

Reply:
So, IIUC, this is for future extension, reading settings from EFI variable, right?

Reply:
I guess so, but this way the setting will stay compatible

Reply:
I think it is better to use `bool` now, and change it later when such feature is implemented. But OK. Not a strong opinion.